tests/test_deprecation.py

```diff
@@ -83,6 +82,10 @@ def test_interfaces() -> None:
 
 def test_load_and_save_results() -> None:
     """Test that load_results and save_results are deprecated."""
+    # TODO 3.0: Remove this test.
+    # pylint: disable=import-outside-toplevel
+    from pylint.config import load_results, save_results
```

Comment:
Is this necessary to pass? All of these tests will be removed in `3.0`, as per test file name 😄

Reply:
Not strictly but the CI pipeline is half done 😄